config/locales/en.yml

```diff
@@ -66,3 +66,11 @@ en:
     users: "Users:"
     newuser: "New user"
     delete: "Delete"
+
```

Comment:
Are you sure we need to store these symbols in translations? I think these will be the same for all languages. May we put it as constant?